doc/aurbuild.md

```diff
@@ -0,0 +1,27 @@
+# aurbuild
+
+## Name
+
+aurbuild - Build packages to a local repository
+
+## Synopsis
+
+aurbuild [+-a ARG] [+-cd ARG] [+-p ARG] [+-r ARG} [--] ARGS...
+
+## Description
+
+Build the packages from a PKGBUILD and add them to a local repository. The PKGBUILD file must be given, just like the name of the database and the directories for the packages and the database files.
```

Comment:
Well, that's odd. Could be that it reads the directory name from the first line (pkgname).